openshift-hack/images/os/Dockerfile

```diff
@@ -1,11 +1,9 @@
-# fedora:29 is an image built within the release scripts:
-# https://github.com/openshift/release/blob/b45a09d248b8cdb8fe3bf5f3cfa0b4fee57d04c8/ci-operator/config/openshift/kubernetes/openshift-kubernetes-release-4.10.yaml#L63-L65
-FROM fedora:29 AS build
```

Comment:
I think part of the problem here may be that the Prow jobs are trying to substitute this at runtime...so what we really need to do is change the image reference in openshift/release, not here.  I think around here https://github.com/openshift/release/blob/b45a09d248b8cdb8fe3bf5f3cfa0b4fee57d04c8/ci-operator/config/openshift/kubernetes/openshift-kubernetes-release-4.10.yaml#L6

Reply:
right o/release needs to be updated to remove all of the f:29 changes -> the centos which is not yet available. not sure how long anything takes to populate tho.

Reply:
It looks like it's there:
```
$ oc -n openshift get is | grep -i stream
centos                                                registry.ci.openshift.org/openshift/centos                                                stream8,stream9,7                                                7 hours ago
$
```

Try https://github.com/openshift/release/pull/32366 ?